server/get_payload.go

```diff
@@ -257,6 +258,184 @@ func (m *BoostService) getPayload(log *logrus.Entry, signedBlindedBeaconBlockByt
 	return <-resultCh, originalBid
 }
 
+// getPayloadV2 submits the signed blinded beacon block to relays for submission without returning the payload and blobs
+func (m *BoostService) getPayloadV2(log *logrus.Entry, signedBlindedBeaconBlockBytes []byte, userAgent, proposerContentType, proposerAcceptContentTypes, proposerEthConsensusVersion string) (bool, bidResp) {
```

Comment:
I think the only difference b/w `getPayload` and `getPayloadV2` is the API call right? Can we avoid code duplication in this case and have a `innerGetPayload` method which takes in a parameter which decides whether to call a v1 api or v2 api? that would make the code much cleaner.
Sth like:

`func (m *BoostService) innerGetPayload(log *logrus.Entry, signedBlindedBeaconBlockBytes []byte, userAgent, proposerContentType, proposerAcceptContentTypes, proposerEthConsensusVersion string, apiType string) (bool bidResp)`